Uno.Gallery/Uno.Gallery.csproj

```diff
@@ -127,6 +127,7 @@
 			<PrivateAssets>all</PrivateAssets>
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
 		</PackageReference>
-		<EmbeddedResource Include="Platforms\WebAssembly\WasmScripts\*.js" />
+		<EmbeddedResource Include="Platforms\WebAssembly\WasmScripts\UnoGallery.js" />
```

Comment:
Is there a reason to remove the globbing here?

BTW, this should be handle by Uno.Sdk, no? @dansiegel?

Reply:
TS is compiled to JS in a later target so globbing isn't going to pick the not yet existing file